egs/librispeech/asrtts/run.sh

```diff
@@ -0,0 +1,596 @@
+            ark,scp:${outdir}_denorm/${name}/feats.ark,${outdir}_denorm/${name}/feats.scp
+        convert_fbank.sh --nj ${nj} --cmd "${train_cmd}" \
+            --fs ${fs} \
+            --fmax "${fmax}" \
+            --fmin "${fmin}" \
+            --n_fft ${n_fft} \
+            --n_shift ${n_shift} \
+            --win_length "${win_length}" \
+            --n_mels ${n_mels} \
+            ${outdir}_denorm/${name} \
+            ${outdir}_denorm/${name}/log \
+            ${outdir}_denorm/${name}/wav
+    done
+    fi
+fi
+
+if [ ${stage} -le 5 ] && [ ${stop_stage} -ge 5 ]; then
+    echo "stage 5: ASR-TTS training, decode and synthesize"
+    asrttsexpdir=exp/asrtts_${tag}
+    train_opts=
+    sample_topk=0
+    sample_scaling=0.1
+    teacher_weight=0.1
+    n_samples=5
+
+    if [ ! -s  ${feat_tr_up_dir}/data_rnd.json ]; then
+        bash utils/copy_data_dir.sh data/${train_unpaired_set} data/${train_unpaired_set}_rnd
+        bash local/rand_datagen.sh --jsonout "${feat_tr_up_dir}/data_rnd.json" \
+            --nlsyms $nlsyms --dict $dict --xvec ${spk_vector}/xvectors_$train_set \
+            $feat_tr_p_dir $feat_tr_up_dir data/${train_unpaired_set}_rnd
+    fi
+    if [ $unpair == 'dual' ]; then
+        tr_json_list="${feat_tr_up_dir}/data_rnd.json"
+    elif [ $unpair == 'dualp' ]; then
+       tr_json_list="${feat_tr_up_dir}/data_rnd.json ${feat_tr_p_dir}/data.json"
+    else
+        tr_json_list="${feat_tr_p_dir}/data.json"
+    fi
+    if [ "$policy_gradient" = "true" ]; then
+        asrttsexpdir=${asrttsexpdir}_exploss_pgrad
+        train_opts="$train_opts --policy-gradient"
+    fi
+    if [ "$rnnlm_loss" = "ce" ]; then
+        asrttsexpdir=${asrttsexpdir}_rnnlmloss_${rnnlm_loss}
+        train_opts="$train_opts --rnnlm $rnnlm_model --rnnlm-conf $rnnlm_model_conf --rnnloss ce" 
+    elif [ "$rnnlm_loss" = "kld" ]; then
+        asrttsexpdir=${asrttsexpdir}_rnnlmloss_${rnnlm_loss}
+        train_opts="$train_opts --rnnlm $rnnlm_model --rnnlm-conf $rnnlm_model_conf --rnnloss kld" 
+    elif [ "$rnnlm_loss" = "mmd" ]; then
+        asrttsexpdir=${asrttsexpdir}_rnnlmloss_${rnnlm_loss}
+        train_opts="$train_opts --rnnlm $rnnlm_model --rnnlm-conf $rnnlm_model_conf --rnnloss mmd" 
+    elif [ "$rnnlm_loss" = "kl" ]; then
+        asrttsexpdir=${asrttsexpdir}_rnnlmloss_${rnnlm_loss}
+        train_opts="$train_opts --rnnlm $rnnlm_model --rnnlm-conf $rnnlm_model_conf --rnnloss kl" 
+    fi
+    if [ $asrtts_train == 'true' ]; then
+        ${cuda_cmd} --gpu 1 ${asrttsexpdir}/train.log \
+        asrtts_train.py \
+        --ngpu 6 \
+        --backend ${backend} \
+        --outdir ${asrttsexpdir}/results \
+        --debugmode ${debugmode} \
+        --dict ${dict} \
+        --debugdir ${asrttsexpdir} \
+        --minibatches ${N} \
+        --verbose ${verbose} \
+        --resume ${resume} \
+        --train-json ${tr_json_list} \
+        --valid-json ${feat_dt_dir}/data.json \
+        --etype ${etype} \
```

Comment:
It is possible to use config for asrtts as well?
This makes your `run.sh` clean.